tests/Core/Content/Product/SalesChannel/Listing/ProductListingFeaturesSubscriberTest.php

```diff
@@ -1193,11 +1195,17 @@ public function testMultipleListingEventsWillOnlyHandleTheRequestOnce(): void
         $decoratedSubscriberMock = $this->getMockBuilder(ShopwareProductListingFeaturesSubscriber::class)
             ->disableOriginalConstructor()
             ->getMock();
-        $decoratedSubscriberMock->expects($this->once())->method('handleListingRequest');
+        $decoratedSubscriberMock->expects($this->exactly(4))->method('handleListingRequest');
 
-        $subscriber = $this->getProductListingFeaturesSubscriber([
-            ShopwareProductListingFeaturesSubscriber::class => $decoratedSubscriberMock
-        ]);
+        $findologicSearchServiceMock = $this->getMockBuilder(FindologicSearchService::class)
+            ->disableOriginalConstructor()
+            ->getMock();
+        $findologicSearchServiceMock->expects($this->once())->method('doNavigation');
```

Comment:
Please move this above the `$decoratedSubscriberMock`, so it is immediately clear that this is what we're really interested in the test.